openvdb/openvdb/openvdb.cc

```diff
@@ -20,25 +20,6 @@
     #error ABI <= 5 is no longer supported
 #endif
 
-// If using an OPENVDB_ABI_VERSION_NUMBER that has been deprecated, issue an error
-// directive. This can be optionally suppressed by setting the CMake option
-// OPENVDB_USE_DEPRECATED_ABI_<VERSION>=ON.
-#ifndef OPENVDB_USE_DEPRECATED_ABI_6
-    #if OPENVDB_ABI_VERSION_NUMBER == 6
-        #error ABI = 6 is deprecated, CMake option OPENVDB_USE_DEPRECATED_ABI_6 suppresses this error
-    #endif
-#endif
-
-// If using a future OPENVDB_ABI_VERSION_NUMBER, issue an error directive.
-// This can be optionally suppressed by setting the CMake option
-// OPENVDB_USE_FUTURE_ABI_<VERSION>=ON.
-#ifndef OPENVDB_USE_FUTURE_ABI_9
-    #if OPENVDB_ABI_VERSION_NUMBER == 9
-        #error ABI = 9 is still in active development and has not been finalized, \
-CMake option OPENVDB_USE_FUTURE_ABI_9 suppresses this error
-    #endif
-#endif
```

Comment:
These errors are redundant. The CMake build system explicitly errors if they are not defined

Reply:
If it's not a big deal, I would prefer to leave these here for safety. I agree with CMake being our official and only supported build system, but I do build the VDB core library without CMake.

Reply:
Makes sense, will add them back with a comment